### DISCUSSION TITLE
Sound bug

### DISCUSSION THREAD

**Discussion by theviper22** [obsproject/obs-studio#11368] [General]
when using the internal navigator the sound is dulled, it sounds bad, sometimes it is not even heard and there is an annoying tingling, I use it to play chess games and the movements are not of quality, when the movements are constant there is no problem, but if there is a difference between 5 or more seconds between movement the problem begins.

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
